images/postgres/tests/02-query.sh

```diff
@@ -4,7 +4,7 @@ set -o errexit -o nounset -o errtrace -o pipefail -x
 
 CONTAINER_NAME=${CONTAINER_NAME:-"postgres-${FREE_PORT}"}
 
-docker run --network=host -e POSTGRES_PASSWORD=password -d --name $CONTAINER_NAME $IMAGE_NAME
+docker run -e POSTGRES_PASSWORD=password -d --name $CONTAINER_NAME $IMAGE_NAME
 trap "docker logs $CONTAINER_NAME && docker rm -f $CONTAINER_NAME" EXIT
 sleep 5
-docker run --network=host --entrypoint=/usr/bin/psql $IMAGE_NAME -U postgres -h 127.0.0.1 -l -t --csv | head -n 1 | grep postgres
+docker exec $CONTAINER_NAME /usr/bin/psql -U postgres -h 127.0.0.1 -l -t | head -n 1 | grep postgres
```

Comment:
`--csv` was introduced in `psql` version 12, so dropping this (if I can) to support using this same test with version 11 🤞